tests/test_group_method.py

```diff
@@ -27,54 +27,46 @@ def test_group_random1():
         "Robert",
     ]
     lst2 = ["Dan", "Angie", "Austin", "Izaak", "Nick", "Jeff"]
-    group_size = 3
-    group_size2 = 2
     num_group = 2
     num_group2 = 3
-    actual_output = group_creation.group_random_group_size(lst, group_size)
-    actual_output2 = group_creation.group_random_group_size(lst2, group_size2)
     actual_output3 = group_creation.group_random_num_group(lst, num_group)
     actual_output4 = group_creation.group_random_num_group(lst2, num_group2)
-
-    assert len(actual_output) == 4
-    assert len(actual_output[0]) == 3
-    assert len(actual_output2) == 3
-    assert len(actual_output2[0]) == 2
     assert len(actual_output3) == 2
     assert len(actual_output3[0]) == 6
     assert len(actual_output4) == 3
     assert len(actual_output4[0]) == 2
 
 
-@given(group_size=integers(min_value=1, max_value=3))
-@settings(verbosity=Verbosity.verbose, deadline=None)
-@pytest.mark.hypothesisworks
-def hypothesis_test_group_random1(group_size):
-    """this hypothesis test can generate the group numbers and test if it pass
-        the requirements"""
-    lst = [
-        "Austin",
-        "Dan",
-        "Angie",
-        "Cullen",
-        "Chase",
-        "Vinny",
-        "Nick",
-        "Jeff",
-        "James",
-        "Kelly",
-        "Nikki",
-        "Robert",
-    ]
-    lst2 = ["Dan", "Angie", "Austin", "Izaak", "Nick", "Jeff"]
-    size_count = group_size
-    actual_output = group_creation.group_random_group_size(lst, group_size)
-    actual_output2 = group_creation.group_random_group_size(lst2, group_size)
-
-    assert len(actual_output) == 12 // size_count
-    assert len(actual_output[0]) == size_count
-    assert len(actual_output2) == 6 // size_count
-    assert len(actual_output2[0]) == size_count
+# Test uses now deleted group_size dependent functions, must be rewritten
+# @given(group_size=integers(min_value=1, max_value=3))
+# @settings(verbosity=Verbosity.verbose, deadline=None)
+# @pytest.mark.hypothesisworks
+# def hypothesis_test_group_random1(group_size):
+#     """this hypothesis test can generate the group numbers and test if it pass
+#         the requirements"""
+#     lst = [
+#         "Austin",
+#         "Dan",
+#         "Angie",
+#         "Cullen",
+#         "Chase",
+#         "Vinny",
+#         "Nick",
+#         "Jeff",
+#         "James",
+#         "Kelly",
+#         "Nikki",
+#         "Robert",
+#     ]
+#     lst2 = ["Dan", "Angie", "Austin", "Izaak", "Nick", "Jeff"]
+#     size_count = group_size
+#     actual_output = group_creation.group_random_group_size(lst, group_size)
+#     actual_output2 = group_creation.group_random_group_size(lst2, group_size)
+#
+#     assert len(actual_output) == 12 // size_count
+#     assert len(actual_output[0]) == size_count
+#     assert len(actual_output2) == 6 // size_count
+#     assert len(actual_output2[0]) == size_count
```

Comment:
Is this suppose to be commented out?

Reply:
Where I could not change them or did not know whether I should, I commented them out and left a note in the comments

Reply:
Nvm, I didn't see that part in the description.